model/user.go

```diff
@@ -50,6 +53,13 @@ type User struct {
 	LastPictureUpdate  int64     `json:"last_picture_update"`
 }
 
+type GitLabUser struct {
```

Comment:
instead of muddling user.go wonder if these service specific users should be in their own files.  especially if we start adding a bunch more like gmail, outlook.com, salesforce, etc.